eva-accession-release-automation/publish_release_to_ftp/publish_release_files_to_ftp.py

```diff
@@ -275,30 +281,29 @@ def publish_species_level_files_to_ftp(release_properties, species_current_relea
 def publish_release_top_level_files_to_ftp(release_properties):
     grep_command_for_release_level_files = "grep " + " ".join(['-e "{0}"'.format(filename) for filename in
                                                                release_top_level_files_to_copy])
-    run_command_with_output("Copying release level files from {0} to {1}...."
-                            .format(release_properties.staging_release_folder,
-                                    release_properties.public_ftp_current_release_folder),
-                            "(find {0} -maxdepth 1 -type f | {1} | xargs -i rsync -av {{}} {2})".format(
-                                release_properties.staging_release_folder, grep_command_for_release_level_files,
-                                release_properties.public_ftp_current_release_folder))
+    run_command_with_output(f"""Copying release level files from {release_properties.staging_release_folder} to 
+                                {release_properties.public_ftp_current_release_folder}....""",
+                            f"""(find {release_properties.staging_release_folder} -maxdepth 1 -type f | 
+                            {grep_command_for_release_level_files} | 
+                            xargs -i rsync -av {{}} {release_properties.public_ftp_current_release_folder})""")
 
 
 def create_requisite_folders(release_properties):
     run_command_with_output("Creating by_species folder for the current release...",
-                            "mkdir -p " + os.path.join(release_properties.public_ftp_current_release_folder,
-                                                       by_species_folder_name))
+                            f"""mkdir -p {os.path.join(release_properties.public_ftp_current_release_folder,
+                                                       by_species_folder_name)}""")
```

Comment:
These are multiline, so requires """

Reply:
oh, I thought these didn't need `"""` because the line break is technically within the `os.path.join`, but now that I actually try it I see I am wrong... ignore me!